src/main/java/sirius/biz/codelists/LookupValues.java

```diff
@@ -87,4 +89,21 @@ public LookupTable getTable() {
     public LookupValue.CustomValues getCustomValues() {
         return customValues;
     }
+
+    public String getTableName() {
+        return lookupTableName;
+    }
+
+    /**
+     * Resolves a String to present to the User for each value according to {@link #display}
+     *
+     * @return a map of the value codes, along with their display Strings
+     */
+    public Map<String, String> resolveDisplayStrings() {
+        Map<String, String> codeAndDisplays = new HashMap<>();
+        data().forEach(code -> {
+            codeAndDisplays.put(code, display.resolveDisplayString(getTable(), code));
+        });
+        return codeAndDisplays;
```

Comment:
`return data().stream().map(code -> Tuple.create(code, isplay.resolveDisplayString(getTable(), code)).collect(Collectors.toList());` or the like